app/models/spree/line_item_decorator.rb

```diff
@@ -131,7 +131,9 @@ def display_included_tax
     Spree::Money.new(included_tax, currency: currency)
   end
 
-  delegate :display_name, to: :variant
+  def display_name
+    variant.display_name
+  end
```

Comment:
:hankey: we don't need this change. It simply changes the RuntimerError to a NilClass error